src/patterns/validation/index.md.njk

```diff
@@ -39,14 +39,14 @@ Validation should refuse to accept:
 
 For example, if you’re asking for someone’s date of birth you should not accept ‘13’ in the month field.
 
-Validation should help remove (or ignore) any spaces or invisible characters, unless it makes the information too ambiguous for you to use.
+Use validation to ignore, or remove, unwanted characters, unless it would make the information too unclear for you to use. For example, any spaces, invisible characters or punctuation, like hyphens, brackets, dashes and full stops.
 
-If spaces or invisible characters cause a validation error, it'd be difficult for the user to see and fix — especially if they were added by accident.
+If these characters caused a validation error, it would be difficult for the user to see and fix — especially if added by accident.
 
-Consider removing spaces and invisible characters entered:
+You should remove unwanted characters entered:
```

Comment:
Could you also ignore them? If so, should probably change this to "You should ignore or remove..."

Reply:
```suggestion
You should ignore or remove unwanted characters entered:
```